src/main.rs

```diff
@@ -743,21 +744,49 @@ async fn send(args: SendArgs) -> anyhow::Result<()> {
 
     #[cfg(feature = "clipboard")]
     {
-        use console::{Key, Term};
+        use crossterm::event::{Event, EventStream, KeyCode, KeyEvent, KeyEventKind, KeyModifiers};
+        use nix::{
+            sys::signal::{kill, Signal},
+            unistd::Pid,
+        };
 
         // Add command to the clipboard
         if args.clipboard {
             add_to_clipboard(&ticket);
         }
 
         let _keyboard = tokio::task::spawn(async move {
-            let term = Term::stdout();
             println!("press c to copy command to clipboard, or use the --clipboard argument");
-            loop {
-                if let Ok(Key::Char('c')) = term.read_key() {
-                    add_to_clipboard(&ticket);
-                }
-            }
+
+            // `enable_raw_mode` will remember the current terminal mode
+            // and restore it when `disable_raw_mode` is called.
+            enable_raw_mode().unwrap_or_else(|err| eprintln!("Failed to enable raw mode: {err}"));
+            let event_stream = EventStream::new();
+            event_stream
+                .for_each(move |e| match e {
+                    Err(err) => eprintln!("Failed to process event: {err}\r"),
+                    // c is pressed
+                    Ok(Event::Key(KeyEvent {
+                        code: KeyCode::Char('c'),
+                        modifiers: KeyModifiers::NONE,
+                        kind: KeyEventKind::Press,
+                        ..
+                    })) => add_to_clipboard(&ticket),
+                    // Ctrl+c is pressed
+                    Ok(Event::Key(KeyEvent {
+                        code: KeyCode::Char('c'),
+                        modifiers: KeyModifiers::CONTROL,
+                        kind: KeyEventKind::Press,
+                        ..
+                    })) => {
+                        disable_raw_mode()
+                            .unwrap_or_else(|e| eprintln!("Failed to disable raw mode: {e}\r"));
+                        kill(Pid::from_raw(0), Some(Signal::SIGINT))
+                            .unwrap_or_else(|e| eprintln!("Failed to end process: {e}"));
+                    }
```

Comment:
This is where Windows build would fail since I tried to trigger a UNIX signal using UNIX `kill` syscall.

Reply:
Now it will compile, but I wonder whether it works correctly on Windows.

Reply:
why do you need to trigger `kill` can't you simply exit the process?

Reply:
This matches the original `read_key` behavior in `console` crate:

https://docs.rs/console/latest/src/console/unix_term.rs.html#351-362

```rust
    // if the user hit ^C we want to signal SIGINT to ourselves.
    if let Err(ref err) = rv {
        if err.kind() == io::ErrorKind::Interrupted {
            if !ctrlc_key {
                unsafe {
                    libc::raise(libc::SIGINT);
                }
            } else {
                return Ok(Key::CtrlC);
            }
        }
    }
```